app/bundles/EmailBundle/Views/Email/details.html.twig

```diff
@@ -0,0 +1,353 @@
+            'primary'   : true,
+        } %}
+    {% endif %}
+{% endif %}
+
+{# Only show A/B test button if not already a translation of an a/b test #}
+{% set allowAbTest = email.isTranslation(true) and translations['parent'].isVariant(true) ? false : true %}
+{% if not isEmbedded %}
+    {% block actions %}
+        {{- include('MauticCoreBundle:Helper:page_actions.html.twig', {
+            'item'            : email,
+            'templateButtons' : {
+                'edit' : securityHasEntityAccess(
+                    permissions['email:emails:editown'],
+                    permissions['email:emails:editother'],
+                    email.getCreatedBy(),
+                ),
+                'clone'  : permissions['email:emails:create'],
+                'abtest' : (allowAbTest and permissions['email:emails:create']),
+                'delete' : securityHasEntityAccess(
+                    permissions['email:emails:deleteown'],
+                    permissions['email:emails:deleteother'],
+                    email.getCreatedBy()
+                ),
+                'close' : securityHasEntityAccess(
+                    permissions['email:emails:viewown'],
+                    permissions['email:emails:viewother'],
+                    email.getCreatedBy()
+                ),
+            },
+            'routeBase'     : 'email',
+            'customButtons' : customButtons,
+        }) -}}
+    {% endblock %}
+    {% block %}
+        {{- include('MauticCoreBundle:Helper:publishstatus_badge.html.twig', {
+            'entity' : email
+        }) -}}
+    {% endblock %}
+{% endif %}
+
+{# start: box layout #}
+<div class="box-layout">
+    <!-- left section -->
+    <div class="col-md-9 bg-white height-auto">
+        <div class="bg-auto">
+            <!-- email detail header -->
+            <div class="pr-md pl-md pt-lg pb-lg">
+                <div class="box-layout">
+                    <div class="col-xs-10">
+                        <div>
+                            {{ toHtml(email.getSubject(), 'short') }}
+                        </div>
+                        {% if email.isVariant(true) %}
+                        <div class="small">
+                            <a href="{{ path('mautic_email_action', {'objectAction' : 'view', 'objectId' : variants['parent'].getId()}) }}" data-toggle="ajax">
+                                {{ 'mautic.core.variant_of'|trans ({'%parent%' : variants['parent'].getName()}) }}
+                            </a>
+                        </div>
+                        {% endif %}
+                        {% if email.isTranslation(true) %}
+                        <div class="small">
+                            <a href="{{ path('mautic_email_action', {'objectAction' : 'view', 'objectId' : translations['parent']->getId()}) }}" data-toggle="ajax">
+                                {{ 'mautic.core.translation_of'|trans ({'%parent%' : $translations['parent'].getName()}) }}
```

Comment:
Twig syntax error. Should be `{{ 'mautic.core.translation_of'|trans({'%parent%' : translations['parent'].getName()}) }}` assuming that `translations` is a variable